sdk/iot/Azure.Iot.Hub.Service/src/IoTHubServiceClient.cs

```diff
@@ -54,24 +61,91 @@ protected IoTHubServiceClient()
         /// <summary>
         /// Initializes a new instance of the <see cref="IoTHubServiceClient"/> class.
         /// </summary>
-        public IoTHubServiceClient(Uri endpoint)
-            : this(endpoint, new IoTHubServiceClientOptions())
+        /// <param name="connectionString">
+        /// The IoT Hub connection string, with either "iothubowner", "service", "registryRead" or "registryReadWrite" policy, as applicable.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#access-control-and-permissions"/>.
+        /// </param>
+        public IoTHubServiceClient(string connectionString)
+            : this(connectionString, new IoTHubServiceClientOptions())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IoTHubServiceClient"/> class.
+        /// </summary>
+        /// <param name="connectionString">
+        /// The IoT Hub connection string, with either "iothubowner", "service", "registryRead" or "registryReadWrite" policy, as applicable.
+        /// For more information, see <see href="https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-security#access-control-and-permissions"/>.
+        /// </param>
+        /// <param name="options">
+        /// Options that allow configuration of requests sent to the IoT Hub service.
+        /// </param>
+        public IoTHubServiceClient(string connectionString, IoTHubServiceClientOptions options)
```

Comment:
is it mandatory to provide options?

Reply:
No, we do have a constructor overload that doesn't take in `IoTHubServiceClientOptions`.

Reply:
nm, missed seeing it.